src/bin/pg_autoctl/pgctl.c

```diff
@@ -500,6 +500,11 @@ prepare_guc_settings_from_pgsetup(const char *configFilePath,
 								  setting->name, pgSetup->ssl.serverKey);
 			}
 		}
+		else if (streq(setting->name, "citus.node_conninfo"))
+		{
+			appendPQExpBuffer(config, "%s = '%s'\n",
```

Comment:
there should be all the SSL related connection option in here, i.e. `sslrootcert` and `sslcrl` (maybe more)

Reply:
Done now, good point.